charts/jenkins/templates/jenkins-master-secondary-ingress.yaml

```
@@ -0,0 +1,40 @@
+{{- if .Values.master.secondaryingress.enabled }}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: {{ .Values.master.secondaryingress.apiVersion }}
+{{- end }}
+kind: Ingress
+metadata:
+  namespace: {{ template "jenkins.namespace" . }}
+  labels:
+    "app.kubernetes.io/name": '{{ template "jenkins.name" .}}'
+    "helm.sh/chart": "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    "app.kubernetes.io/managed-by": "{{ .Release.Service }}"
+    "app.kubernetes.io/instance": "{{ .Release.Name }}"
+    "app.kubernetes.io/component": "{{ .Values.master.componentName }}"
+{{- if .Values.master.secondaryingress.labels }}
```

Would be nice to indent if statements so that it's easier to read.

---

```suggestion
    {{- if .Values.master.secondaryingress.labels }}
```
so like that?

or intend everything one level in more?